root/etc/s6-overlay/s6-rc.d/svc-homeassistant/run

```diff
@@ -1,8 +1,9 @@
 #!/usr/bin/with-contenv bash
+# shellcheck shell=bash
 
 setcap 'cap_net_bind_service=+ep' /usr/bin/python3.10
 
 exec \
     s6-notifyoncheck -d -n 60 -w 5000 -c "nc -z 127.0.0.1 8123" \
```

Comment:
Should we use `localhost` here to keep it consistent with the other images?

Reply:
Resolved in https://github.com/linuxserver/docker-homeassistant/pull/65/commits/6eebc2ae3bbe7917767bac149057c31a4e591738